clara/py_parser.py

```diff
@@ -74,6 +74,20 @@ def visit_Module(self, node):
 
     # Methods for visiting literals
 
+    def visit_Constant(self, node):
```

Comment:
Again, it would be cool to add some tests for this!

Reply:
I take a look at tests and try to add more. In some cases, like this one, it is not so straightforward, and may take me time to do that. Unfortunately I'm a little busy these days.

Reply:
I'd add very simple tests that just parses & evaluates (using interpreter) given expressions.

If you're not able to do this in some time then I might finish these PRs once I have time.

Reply:
It's very good if you can handle tests yourself. I've been far from code for a while.

Reply:
Will add some tests #25.